scripts/generate-package-json.js

```diff
@@ -0,0 +1,8 @@
+var fs = require('fs');
```

Comment:
Since is a little weird IMO we should add a comment explaining what this file does and why